### PR TITLE
feat: implement save on blur with debounce 5 seconds

### PR DESCRIPTION
**Pull request by mmehmetAliIzci**
# Changes

## Related issues

https://github.com/FrontMen/fm-hours/issues/172

## Added

Now with debounce of 5 seconds, any changes are autosaved in timesheet edits.

## How to test

Start editing hours, kms and comment messages. It should trigger an autosave after 5 seconds

### CODE REVIEW

**Review by vladPinteaFrontmen — COMMENTED** [review]
Looks good. 

I reckon the comment/message should also trigger an autosave, not just the hours/km inputs

File: pages/records.vue [hunk 1]
```diff
@@ -54,7 +55,7 @@
             />
           </template>
         </weekly-timesheet>
-
+
```
Comment (vladPinteaFrontmen) [inline pages/records.vue:58]: nitpick: remove spaces

**Review by vladPinteaFrontmen — APPROVED** [review]
fix the nitpick and feel free to merge

File: pages/records.vue (hunk 1 above)
Comment (mmehmetAliIzci) [reply pages/records.vue]: Done

**Pull request merged**